bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v19/V19BlockFilterRpc.scala

```diff
@@ -0,0 +1,39 @@
+package org.bitcoins.rpc.client.v19
+
+import org.bitcoins.core.crypto.DoubleSha256DigestBE
+import org.bitcoins.core.gcs.{BlockFilter, FilterType}
+import org.bitcoins.rpc.client.common.Client
+import org.bitcoins.rpc.jsonmodels.GetBlockFilterResult
+import org.bitcoins.rpc.serializers.JsonReaders.DoubleSha256DigestBEReads
+import play.api.libs.json._
+
+import scala.concurrent.Future
+
+/**
+  * Gets the BIP158 filter for the specified block.
+  * This RPC is only enabled if block filters have been created using the -blockfilterindex configuration option
+  * @see [[https://bitcoincore.org/en/doc/0.19.0/rpc/blockchain/getblockfilter]]
+  */
+trait V19BlockFilterRpc {
+  self: Client =>
+
+  private case class TempBlockFilterResult(
```

Comment:
Why is this needed? Can we add a comment here indicating why we need this temp result?